### PR TITLE
Extract choose_from_list_without_replacement to util

### PR DESCRIPTION
**Pull request by dvadym**

### CODE REVIEW

File: pipeline_dp/utils.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import numpy as np
+
+
+def choose_from_list_without_replacement(a: list, size: int) -> list:
+    if len(a) <= size:
+        return a
+    # np.random.choice makes casting of elements to numpy types
+    # which is undesirable by 2 reasons:
+    # 1. Apache Beam can not serialize numpy types.
+    # 2. It might lead for losing precision (e.g. arbitrary
```
Comment (miracvbasaran) [inline pipeline_dp/utils.py:24]: nit: `It might lead to a loss of precision...`

File: pipeline_dp/dp_engine.py [hunk 2]
```diff
@@ -12,16 +12,15 @@
...
 import dataclasses
 import functools
 from typing import Any, Callable, Tuple
-import numpy as np
 
 import pipeline_dp
 from pipeline_dp import combiners
 import pipeline_dp.contribution_bounders as contribution_bounders
 import pipeline_dp.report_generator as report_generator
+import pipeline_dp.utils as utils
```
Comment (miracvbasaran) [inline pipeline_dp/dp_engine.py:23]: I don't feel great about creating a utils module, maybe we can be more clear about what we are refactoring here. Do we expect to refactor away more statistics/probability-related functions? Then, maybe we can rename it to something like `stats_helpers`/`stats_utils`? If we don't expect any more utils, maybe we can just keep it in DPEngine?

**Review by miracvbasaran — CHANGES_REQUESTED** [review]

File: pipeline_dp/dp_engine.py (hunk 2 above)
Comment (dvadym) [reply pipeline_dp/dp_engine.py:23]: Thanks, good point! I don't expect now other functions. But there will be call of this function in contribution_bounder.py in the next PR (WIP).
I've renamed to sampling_utils.py, which is much better represents what this code doing

**Review by dvadym — COMMENTED** [review]
Thanks for comments! PTAL

File: tests/sampling_utils_test.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+from absl.testing import absltest
```
Comment (miracvbasaran) [inline tests/sampling_utils_test.py:1]: No license header here?

**Review by miracvbasaran — APPROVED** [review]

File: tests/sampling_utils_test.py (hunk 3 above)
Comment (dvadym) [reply tests/sampling_utils_test.py:15]: Thanks, added.

**Review by dvadym — COMMENTED** [review]
Thanks for review!

**Pull request merged**